app/models/job_invocation_composer.rb

```diff
@@ -282,20 +282,11 @@ def template_invocation_input_value_for(input)
   end
 
   def job_template_ids
-    job_invocation.template_invocations.map(&:template_id)
+    job_invocation.pattern_template_invocations.map(&:template_id)
   end
 
   private
 
-  def dup_template_invocations(job_invocation)
```

Comment:
replaced by existing deep_clone (that no longer save records), effective_user was useless anyway since dup copied that value as well